app/(dashboard)/dashboard/interviews/_components/ExportInterviewsDialog.tsx

```diff
@@ -84,6 +85,11 @@ export const ExportInterviewsDialog = ({
       download(url, result.data.name);
       // clean up the URL object
       URL.revokeObjectURL(url);
+
+      // Delete the zip file from UploadThing after 3 seconds
+      setTimeout(async () => {
```

Comment:
What happens if the download takes longer than 3 seconds? Will the asset be deleted before it finishes downloading?

Can we somehow programmatically detect when it has been downloaded, and call the delete function then instead?

Reply:
Okay, I did some research on it. Turns out, it doesn't matter if the zip file gets deleted before a user downloads it: **Once you have a Blob object, it means you have the file data locally in your browser's memory. A Blob object represents a chunk of bytes that holds data of a file.** , so I added a check for the UploadThing response to make sure it doesn't fail.

Reply:
Where did you remove the timeout? I don't see a commit for it.

Reply:
> Where did you remove the timeout? I don't see a commit for it.

Here's the commit : 
https://github.com/complexdatacollective/Fresco/pull/113/commits/b62da1a4ba5b8d47d6ece161e15b2ed36638cb98